apps/src/templates/CodeWorkspace.jsx

```diff
@@ -294,6 +289,23 @@ const styles = {
     padding: 5,
     opacity: 0.9,
     position: 'relative'
+  },
+  chevronButton: {
```

Comment:
Out of curiosity, where was all this styling set previously?

Reply:
Some of it was in `chevron`, above, which I renamed to make more clear. The rest is so a button does not have default button styling (such as a gray background and extra padding)